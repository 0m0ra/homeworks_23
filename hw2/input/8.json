{
    "user": {
        "region": "Saint-Petersburg",
        "registered": "2012-12-24",
        "last_login": "2023-10-01",
        "email": "user@yandex.ru",
        "age": 26
    },
    "user2": {
        "region": "Arxangelsk",
        "registered": "2017-09-08",
        "last_login": "2023-08-09",
        "email": "user2@gmail.com",
        "age": 48
    }
}