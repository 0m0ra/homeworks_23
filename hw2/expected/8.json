{"status": "TimeDateError", "message": "2077-12-24 in uncorrect time"}